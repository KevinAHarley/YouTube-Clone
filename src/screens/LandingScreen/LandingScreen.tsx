import React from "react";
import { View, Text } from "react-native";

import styles from "./LandingScreen.styles";

const LandingScreen = () => {
  return (
    <View style={styles.container}>
      <Text> Test </Text>
    </View>
  );
};

export default LandingScreen;
