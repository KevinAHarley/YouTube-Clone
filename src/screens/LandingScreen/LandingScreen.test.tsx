import React from "react";
import { render } from "@testing-library/react-native";

import LandingScreen from "./LandingScreen";

describe("Landing Screen", () => {
  it("renders correctly", () => {
    const tree = render(<LandingScreen />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
