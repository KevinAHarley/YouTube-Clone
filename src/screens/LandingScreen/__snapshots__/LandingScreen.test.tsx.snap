// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing Screen renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text>
     Test 
  </Text>
</View>
`;
